rosbag2_cpp/include/rosbag2_cpp/writer.hpp

```diff
@@ -126,6 +130,45 @@ class ROSBAG2_CPP_PUBLIC Writer final
     const std::string & type_name,
     const std::string & serialization_format = "cdr");
 
+  /**
+   * Write a serialized message to a bagfile.
+   * Opposing a call to \sa write without topic metadata, the topic will be created.
```

Comment:
nit: I know this was already on the existing signature - but I found the wording confusing. Maybe this would be clearer:

```suggestion
   * The topic will be created if it has not been created already 
```